www/video/videotest.js

```diff
@@ -1,16 +1,18 @@
+var urlCount = 1;
```

Comment:
I would bring the variable declaration above the first function that uses the variable, but totally a nit pick and not sure if this is just my personal preference.